workflow/scripts/compare_diffexp.py

```diff
@@ -0,0 +1,127 @@
+import pyreadr
+import polars as pl
+import polars.selectors as cs
+import altair as alt
```

Comment:
Just curios, why abbreviate `import polars.selectors as cs`, not `ps`? Probably just a common idiom, but I simply come across this import statement for the first time.

And maybe I should also use this opportunity to put in the caveat that I haven't really used polars in anything, yet. So I think I can give no useful feedback as to idiomatic constructs in these scripts.

Reply:
Same goes for `altair`, which I have only used for a few plots, so don't know in much detail.

Reply:
But I guess if these meta-comparison scripts produce the plots that you want, they work fine. They at least read reasonably, and the column names make sense to me.

Reply:
If I remember correctly, Johannes actually wrote this part of the meta_comparisions to give me a sketch and to try out polars himself for the first time. As far as I know the imports actually came from him and for details we would have to ask him.

Regarding Altair and Polars I can't give any guarantee, but the plots and tables do what we want and expect.

Reply:
OK, thanks for digging this up!